src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -490,7 +490,12 @@ protected void Stop()
             // VSCode sends breakpoint requests with the original filename that doesn't exist anymore.
             try
             {
-                scriptFile = editorSession.Workspace.GetFile(setBreakpointsParams.Source.Path);
+                // When you set a breakpoint in the right pane of a Git diff window on a PS1 file,
```

Comment:
Since we're checking for this prefix in another place, maybe it's time to pull this out into a helper method?